liquidhaskell-boot/src/Language/Haskell/Liquid/Parse.hs

```diff
@@ -39,6 +39,7 @@ import qualified Text.PrettyPrint.HughesPJ              as PJ
 import           Text.PrettyPrint.HughesPJ.Compat       ((<+>))
 import           Language.Fixpoint.Types                hiding (panic, SVar, DDecl, DataDecl, DataCtor (..), Error, R, Predicate)
 import           Language.Haskell.Liquid.GHC.Misc       hiding (getSourcePos)
+import           Liquid.GHC.API.Compat
```

Comment:
Compat could be arranged so this import is mandatory with ghc-9.10.1 as well.

Reply:
Yeah, but how? The problem is that if we import `Compat` (or `Compat (foldl')`) on `develop`, it just results in a "redundant import" warning that is turned into an error.

Reply:
Good point. We would need to do something sneaky like:
https://downloads.haskell.org/ghc/latest/docs/users_guide/exts/rebindable_syntax.html?highlight=implicitprelude#custom-prelude-modules-named-prelude

I don't have a strong opinion at the moment.

Reply:
I do have a strong opinion :), but not a solution, unfortunately.

I wish GHC had a proper way to mark (in the module definition, not at the import site) a module as "does not count towards redundant imports".